mnist/README.md

```diff
@@ -0,0 +1,181 @@
+# Mnist
+
+*This example is a Substra implementation of on [the MNIST example from Keras](https://keras.io/examples/vision/mnist_convnet/), inspired from [Substra's Titanic Example](https://github.com/SubstraFoundation/substra/blob/master/examples/titanic/)*
+
+In this example, we'll see how to setup an objective and how to train algorithms on the MNIST dataset.
+
+You can run this example in Google Colab or JupyterLab with this [Notebook](Mnist_Substra_Example_Notebook.ipynb).
+
+## Prerequisites
+
+In order to run this example, you'll need to:
+
+* use Python 3
+* have [Docker](https://www.docker.com/) installed
+* [install the `substra` cli](https://github.com/SubstraFoundation/substra#install) (supported version: 0.6.0)
+* [install the `substratools` library](https://github.com/substrafoundation/substra-tools) (supported version: 0.6.0)
+* [pull the `substra-tools` docker images](https://github.com/substrafoundation/substra-tools#pull-from-private-docker-registry)
+* have access to a Substra installation ([configure your host to a public node ip](https://doc.substra.ai/getting_started/installation/local_install_skaffold.html#network) or [install Substra on your machine](https://doc.substra.ai/getting_started/installation/local_install_skaffold.html))
```

Comment:
In fact it would be great to add a reference to the compatibility table here to ensure that it works with the cli 0.6.0: https://github.com/SubstraFoundation/substra#compatibility-table